PkFokam-Alumni-Network/CommunityWebsite
# Update: Moved Alumni Card styles to App.css and updated component

## WHY
The Alumni Card component currently has its styles in a separate CSS file. Management requested consolidation of CSS files to improve maintainability and reduce the number of separate style files in the project. Moving these styles to App.css will help centralize our styling approach.

## WHAT
- Moved all Alumni Card styles from separate CSS file to src/App.css
- Updated AlumniCard.html to reference App.css instead of its own CSS file
- Updated the CSS path in AlumniCard.html from "./AlumniCard.css" to "../App.css"
- Removed the separate AlumniCard.css file
- Maintained all existing styling and functionality

## TESTING
To test these changes:
1. Navigate to the Alumni Card component in the browser
2. Verify the glassmorphism effect is still working
3. Check that all styles are correctly applied:
   - Card background is semi-transparent
   - Profile image is circular
   - Text colors and spacing are maintained
   - Button styling is preserved
4. Verify responsive design on different screen sizes
